web/src/main/ui/app/components/flows-new/edit-flow/mastering/matching/ui/add-match-option-dialog.component.html

```diff
@@ -51,10 +51,11 @@ <h1 mat-dialog-title>{{getDialogTitle()}}</h1>
     [matTooltip]="selectedType === 'reduce' ? tooltips.weightReduce : tooltips.weightOption"
     matTooltipPosition="left"
     matTooltipShowDelay="500"
-    matTooltipHideDelay="500"
+    matTooltipHideDelay="500000"
```

Comment:
Should the delay here be 500 ms for consistency? Other than that, everything looks good for me!

Reply:
Thanks for catching that, fixed in my latest commit.